### PR TITLE
[oc_erchef] purge couch, couch darklaunch, and couch dependencies from erchef

### PR DESCRIPTION
**Pull request by marcparadise**
This PR purges couchbeam and its depencies from our system.

### CODE REVIEW

File: src/oc_erchef/apps/chef_objects/priv/depselector_rb/Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'dep_selector', :git => 'git://github.com/opscode/dep-selector', :branch => 'master'
-gem 'erlectricity', :git => 'git://github.com/sdelano/erlectricity', :branch => 'master'
+gem 'dep_selector', :git => 'git://github.com/chef/dep-selector', :branch => 'master'
+gem 'erlectricity', :git => 'git://github.com/marcparadise/erlectricity', :branch => 'mp/gcc-4.9.2-compat-fix'
```
Comment (sdelano) [inline src/oc_erchef/apps/chef_objects/priv/depselector_rb/Gemfile]: I's like to get this merged into "master" before we merge this so we don't forget about it.

Comment (sdelano) [inline src/oc_erchef/apps/chef_objects/priv/depselector_rb/Gemfile]: I've merged it. Thanks for updating the PR with more context.

Comment (marcparadise) [inline src/oc_erchef/apps/chef_objects/priv/depselector_rb/Gemfile]: Ugh, this should actually never have been committed here.  It was a temporary workaround for my local build.  Well - good to have it landed though, I'll repoint this one.

**Comment by marcparadise** [comment]
ping @chef/lob because I forgot to earlier.

**Comment by sdelano** [comment]
:+1:

**Pull request merged**